downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -52,6 +52,15 @@ h| help_text | The help text associated with this field | No
 [discrete]
 == Injector Configuration
 
-You can use the Injector configuration field to take the fields from input configuration field and map them into `extra_vars` that can be sent to ansible-rulebook when running the activation. The Injector currently only supports `extra_vars`. 
+You can use the Injector configuration field to take the fields from the Input configuration field and map them into injector credential types that can be sent to ansible-rulebook when running a rulebook activation. {EDAName} supports the following types of injectors: 
+
+* Environment variables (`env`) - used in source plugins for the underlying package or shared library.
+* Ansible extra variables (`extra_vars`) - used for substitution in the rulebook conditions, actions or source plugin parameters.
+* File-based templating (`file`) - used to create file contents from the credential inputs such as certificates and keys, which might be required by source plugins. File injectors provide a way to deliver these certificates and keys to ansible-rulebook at runtime without having to store these keys and certificates in decision environments. Ansible-rulebook creates temporary files and the file names can be accessed using `eda.filename` variables which are automatically created for you once the files have been created (for instance,  "{{eda.filename.my_cert}}”).
```

Comment:
@jself-sudoku Ansible-rulebook should be ansible-rulebook since that is the name of the executable